src/bc/translate.rs

```diff
@@ -62,10 +62,15 @@ impl<'a, 'tcx> Program<'a, 'tcx> {
         self.krates.entry(def_id).or_insert_with(|| {
             // println!("load function {:?}", def_id);
             let cs = &context.tcx.sess.cstore;
-            let mir = cs.maybe_get_item_mir(context.tcx, def_id).unwrap_or_else(||{
-                panic!("no mir for {:?}", def_id);
-            });
-            Rc::new(context.mir_to_bytecode(&mir))
+
+            if let Some(ref node_id) = context.tcx.map.as_local_node_id(def_id){
```

Comment:
This might be clearer as something like:

```rust
let bc = match node_id {
   Some(node_id) => { ... }
   None => { ... }
};
Rc::new(bc)
```

? I'm not totally sure, but at first I didn't realise this was meant to be returning a value from the function...